src/Symfony/Component/Cache/Adapter/RedisAdapter.php

```diff
@@ -61,7 +61,7 @@ public function __construct($redisClient, $namespace = '', $defaultLifetime = 0)
      * @param string $dsn
      * @param array  $options See self::$defaultConnectionOptions
      *
-     * @throws InvalidArgumentException When the DSN is invalid.
+     * @throws InvalidArgumentException When the DSN is invalid
```

Comment:
I would just revert this change as fabbot is unfortunately broken right now. PHP-CS-Fixer 2 is doing things we don't want.